src/locale/i18n.ts

```diff
@@ -40,6 +41,14 @@ import {useLanguagePrefs} from '#/state/preferences'
  */
 export async function dynamicActivate(locale: AppLanguage) {
   switch (locale) {
+    case AppLanguage.am: {
+      i18n.loadAndActivate({locale, messages: messagesAm})
+      await Promise.all([
+        import('@formatjs/intl-pluralrules/locale-data/am'),
```

Comment:
```suggestion
        import('@formatjs/intl-datetimeformat/locale-data/am'),
        import('@formatjs/intl-pluralrules/locale-data/am'),
```

Following #6742 being merged, could you also add this line please?

This will ensure that the date and time for posts are localized correctly on all platforms. Thanks!